src/contributors/02-local-development.md

```diff
@@ -90,6 +90,16 @@ Use `cargo check` to find compilation errors. To start the Lemmy backend, run `c
 
 After making changes, you need to format the code with `cargo +nightly fmt --all` and run the linter with `./scripts/lint.sh`.
 
+If you'd like to speed up compilation times, you can add the following to `~/.cargo/config.toml` (requires nightly rust).
+
+```toml
+[unstable]
+codegen-backend = true
+
+[build]
+rustflags = ["-Clink-arg=-fuse-ld=lld", "-Zthreads=16"]
```

Comment:
Better to define these things in `~/.cargo/config.toml` so it applies to all projects. [mold](https://github.com/rui314/mold) is supposed to be even faster than lld, though you need to install it separately. You can also use [sccache](https://github.com/mozilla/sccache).

```toml
[build]
rustc-wrapper = "/usr/bin/sccache"

[target.x86_64-unknown-linux-gnu]
linker = "clang"
rustflags = ["-C", "link-arg=-fuse-ld=/usr/bin/mold", "-Z", "threads=16"]
```

Reply:
> Better to define these things in ~/.cargo/config.toml

I do have that in L93.

Nice, didn't know about these ones. I'll add those and update this now.